README.md

```diff
@@ -0,0 +1,56 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse Project
+
+> This is my 2nd day project.
+
+## Built With
+
+-HTML
+-CSS
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Usage
+
+### Deployment
+
+
+
```

Comment:
- [ ] [OPTIONAL] Please consider removing empty sections from your README file so it can be more professional

Reply:
@Ismailco Hey with empty sessions you mean all the part of "getting started"? 
or just with prerequisites, setup, usage, and deployment

Reply:
@wdavidcch, the sections that your not using, you can remove getting started or customize it